frontend/src/metabase/sharing/components/SharingSidebar.jsx

```diff
@@ -117,52 +112,45 @@ const getEditingPulseWithDefaults = (state, props) => {
 const mapStateToProps = (state, props) => ({
   pulseId: getPulseId(state, props),
   pulse: getEditingPulseWithDefaults(state, props),
-  cardPreviews: getPulseCardPreviews(state, props),
   formInput: getPulseFormInput(state, props),
   user: getUser(state),
   initialCollectionId: Collections.selectors.getInitialCollectionId(
     state,
     props,
   ),
-  pulseList: getPulseList(state, props),
 });
 
 const mapDispatchToProps = {
   setEditingPulse,
   updateEditingPulse,
   saveEditingPulse,
   fetchPulseFormInput,
-  fetchPulseCardPreview,
   setPulseArchived: Pulses.actions.setArchived,
   testPulse,
-  onChangeLocation: push,
-  goBack,
-  fetchPulsesByDashboardId,
 };
 
-@User.loadList()
+@Pulses.loadList({
+  query: (state, { dashboard }) => ({ dashboard_id: dashboard.id }),
+})
+@User.loadList({ loadingAndErrorWrapper: false })
```

Comment:
Why turn off the loading wrapper? Is it to avoid a sequence of spinners?

Reply:
Yeah it's a consequence of using multiple loaders on a component. My understanding after [running into this before](https://github.com/metabase/metabase/blob/master/frontend/src/metabase/admin/people/containers/PeopleListingApp.jsx#L34) is that both can't use their loading and error wrapper.